docs/resources/rms_originational_assignment_package.md

```diff
@@ -0,0 +1,103 @@
+---
+subcategory: "Config"
+---
+
+# huaweicloud_rms_originational_assignment_package
+
+Manages RMS organizational assignment package resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_rms_originational_assignment_package" "test" {
+  name         = "assignment_package_name"
+  template_key = "organizational_assignment_package_test.tf.json"
```

Comment:
**.tf.json** is the terraform script, suggest to use the example from [API docs](https://support.huaweicloud.com/api-rms/CreateOrganizationConformancePack.html#section5)

```suggestion
  template_key = "example-001.yaml"
```

Reply:
done